core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -96,8 +96,20 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def useStreamingProcess: Boolean =
     sqlConf.getConfString(TiConfigConst.COPROCESS_STREAMING, "false").toLowerCase.toBoolean
 
-  private def isEnableChunk: Boolean =
-    sqlConf.getConfString(TiConfigConst.ENABLE_CHUNK, "true").toLowerCase.toBoolean
+  // streaming only supports TypeDefault. Even we enable chunk, we should still
+  // use TypeDefault.
+  private def getCodecFormat(): EncodeType = {
+    val codecFormatStr =
+      sqlConf.getConfString(TiConfigConst.CODEC_FORMAT, "chblock").toLowerCase
+    if (useStreamingProcess) {
+      return EncodeType.TypeDefault
+    }
+    codecFormatStr match {
+      case "chunk"   => EncodeType.TypeChunk
+      case "chblock" => EncodeType.TypeCHBlock
```

Comment:
If the store type is tikv, we should not use chblock as the encode type

Reply:
TiKV will return data in `Default` codec. We decode data using codec type in resp.